docs/ngrok-agent/changelog.md

```diff
@@ -4,6 +4,14 @@ title: Agent Changelog
 
 # ngrok Agent Changelog
 
+### ngrok Agent 3.1.2 - \[2023-03-13\]
```

Comment:
@russorat @thebluetiger looking again, Euan described this as a `3.2.0` change. this sort of makes sense to me because of the [deprecation of the `—subdomain` and `—hostname` flags](https://semver.org/#spec-item-7). how do we feel about updating this to 3.2.0?